subscribe.go

```diff
@@ -0,0 +1,82 @@
+package bps
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+	"sync"
+)
+
+var (
+	subReg   = map[string]SubscriberFactory{}
+	subRegMu sync.Mutex
+)
+
+// SubscribeOptions defines subscribe options.
```

Comment:
`// Please note that not every option may be supported by the underlying implementation`

Reply:
@dim don't think we need it at all if we're going to switch to single-message consuming (so single-message ack-ing as well)